Feature ru ana

Stellareign opened the pull request:
Проблема с сохранением готового овнера в БД

Stellareign committed:
Добавлен текстовый файл по текущим вопросам. Можно будет туда записывать свой план.
Добавлен пакет контроллер, может понадобиться в дальнейшем.

Stellareign committed:
Добавлен текстовый файл по текущим вопросам. Можно будет туда записывать свой план.
Добавлен пакет контроллер, может понадобиться в дальнейшем.

Stellareign committed:
Добавлен текстовый файл по текущим вопросам. Можно будет туда записывать свой план.
Добавлен пакет контроллер, может понадобиться в дальнейшем.

Stellareign committed:
Добавлен контроллер Pet

Stellareign committed:
Добавлен контроллер Pet

Stellareign committed:
Добавлен контроллер Pet

Stellareign committed:
Merge pull request #2 from Stellareign/dev

Dev

Stellareign committed:
Исправлен листнер, приложение запускается, чатбот реагирует на первые команды

Stellareign committed:
Исправлен листнер, приложение запускается, чатбот реагирует на первые команды

Stellareign committed:
Исправлен листнер, приложение запускается, чатбот реагирует на первые команды

Stellareign committed:
Исправлен листнер, приложение запускается, чатбот реагирует на первые команды

alkor63 committed:
"Оживил" волонтёра - написал 5+1 методов

alkor63 committed:
"Оживил" волонтёра - написал 5+1 методов

alkor63 committed:
Изменил классы owners (под методы Волонтёра)

alkor63 committed:
Изменил класс ownerReport (под методы Волонтёра)

Stellareign committed:
Подправлены аннотации таблиц в классах, бот реагирует на /start, после него можно выбрать приют, так же можно позвать волонтёра

Stellareign committed:
Попытка сделать меню

Stellareign committed:
Подогнаны БД.
Стратегия update - при первом запуске, затем изменить на validate

Stellareign committed:
Подогнаны БД.
Стратеги update - при первом запуске, затем изменить на validate

Stellareign committed:
Подогнаны БД.
Дозагружен енум PetsSex
Стратегия update - при первом запуске, затем изменить на validate

Stellareign committed:
Подогнаны БД.
Стратеги update - при первом запуске, затем изменить на validate

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RuAna

elizaveta-popova committed:
Сообщения для вывода инфы по приюту

Rumonz1 committed:
Начал делать кнопки

Roman-1901 committed:
Merge remote-tracking branch 'origin/feture-TimurA' into feature-RomanN

# Conflicts:
#	src/main/java/com/ward_n6/entity/BotMessaging.java
#	src/main/java/com/ward_n6/entity/owners/CatOwner.java
#	src/main/java/com/ward_n6/entity/owners/DogOwner.java
#	src/main/java/com/ward_n6/entity/owners/PetOwner.java
#	src/main/java/com/ward_n6/entity/pets/Cat.java
#	src/main/java/com/ward_n6/entity/pets/Dog.java
#	src/main/java/com/ward_n6/entity/pets/Pet.java
#	src/main/java/com/ward_n6/entity/reports/CatOwnerReport.java
#	src/main/java/com/ward_n6/entity/reports/DogOwnerReport.java
#	src/main/java/com/ward_n6/entity/reports/OwnerReport.java
#	src/main/java/com/ward_n6/enums/Buttons.java
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java
#	src/main/resources/application.properties

Roman-1901 committed:
q

Stellareign committed:
Merge remote-tracking branch 'origin/feture-TimurA' into feature-RuAna

Roman-1901 committed:
q

Rumonz1 committed:
revork

Stellareign committed:
Добавлены первые кнопки (Тимур)
(нужно исправить кнопку в приюте для кошек)

Stellareign committed:
Добавлены первые кнопки (Тимур) + паракомментов - в проработке
(нужно исправить кнопку в приюте для кошек)

Rumonz1 committed:
revork keyboard

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RuAna

Roman-1901 committed:
q

alkor63 committed:
Исправлен листнер, приложение запускается, чатбот реагирует на первые команды

 Conflicts:
	src/main/java/com/ward_n6/entity/owners/CatOwner.java
	src/main/java/com/ward_n6/entity/owners/DogOwner.java
	src/main/java/com/ward_n6/entity/owners/PetOwner.java
	src/main/java/com/ward_n6/entity/people/Volunteer.java
	src/main/java/com/ward_n6/entity/reports/OwnerReport.java
 Подтяну в свою ветку изменения Романа

alkor63 committed:
Исправлен листнер, приложение запускается, чатбот реагирует на первые команды

 Conflicts:
	src/main/java/com/ward_n6/entity/owners/CatOwner.java
	src/main/java/com/ward_n6/entity/owners/DogOwner.java
	src/main/java/com/ward_n6/entity/owners/PetOwner.java
	src/main/java/com/ward_n6/entity/people/Volunteer.java
	src/main/java/com/ward_n6/entity/reports/OwnerReport.java
 Подтяну в свою ветку изменения Романа

Roman-1901 committed:
q

alkor63 committed:
Merge branch 'feature-RomanN' into feature-AlexeyK

# Conflicts:
#	src/main/java/com/ward_n6/entity/BotMessaging.java
#	src/main/java/com/ward_n6/entity/owners/CatOwner.java
#	src/main/java/com/ward_n6/entity/owners/DogOwner.java
#	src/main/java/com/ward_n6/entity/owners/PetOwner.java
#	src/main/java/com/ward_n6/entity/pets/Cat.java
#	src/main/java/com/ward_n6/entity/pets/Dog.java
#	src/main/java/com/ward_n6/entity/pets/Pet.java
#	src/main/java/com/ward_n6/entity/reports/CatOwnerReport.java
#	src/main/java/com/ward_n6/entity/reports/DogOwnerReport.java
#	src/main/java/com/ward_n6/entity/reports/OwnerReport.java
#	src/main/java/com/ward_n6/enums/Buttons.java
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java
#	src/main/resources/application.properties

alkor63 committed:
создал сервисные классы для 3 таблиц

Roman-1901 committed:
q

alkor63 committed:
создал сервисные классы для 3 таблиц
Наполнил их CRUD-методами,
использовал эти методы в Волонтёрах

alkor63 committed:
создал сервисные классы для 3 таблиц
Наполнил их CRUD-методами,
использовал эти методы в Волонтёрах
Добавил Волонтёрам еще пару методов

Stellareign committed:
Кнопки вынесены в отдельные методы

alkor63 committed:
Допилил Волонтёров и все связанные с ними классы и методы

alkor63 committed:
Написал PetController для Pet

alkor63 committed:
Написал PetController для Pet
Аналогично OwnerController для Owner

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RomanN' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java
#	src/main/resources/application.properties

Stellareign committed:
Merge remote-tracking branch 'origin/feature-AlexeyK' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/entity/owners/PetsOwner.java
#	src/main/java/com/ward_n6/entity/owners/PetsOwnerArchive.java
#	src/main/java/com/ward_n6/service/OwnerService.java

Stellareign committed:
Загружены контроллеры и методы Алексея (взаимодействие со сваггером) -> в доработку

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RuAna

Rumonz1 committed:
test

elizaveta-popova committed:
Сообщения для вывода инфы по приюту ver2

Stellareign committed:
Merge branch 'feature-Elizaveta' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/entity/shelters/CatsShelter.java

Rumonz1 committed:
Merge branch 'feature-RuAna' into feture-TimurA

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java
#	src/main/resources/application.properties

Rumonz1 committed:
Merge branch 'feature-Elizaveta' into feture-TimurA

# Conflicts:
#	src/main/java/com/ward_n6/entity/shelters/CatsShelter.java

Rumonz1 committed:
buttons v2

alkor63 committed:
Написал Exceptions для Pet

alkor63 committed:
Написал Exceptions для Pet (package Service & Controllers)

alkor63 committed:
Написал Exceptions для Owner (package Service & Controllers)

alkor63 committed:
Написал Exceptions для PetsOwner (package Service)

alkor63 committed:
Написал Exceptions для PetsOwnerArchiveServiceImpl (package Service)

alkor63 committed:
Разобрался с Exceptions у Volunteer

Stellareign committed:
Пытаюсь сделать отчё ти персона

Stellareign committed:
Пытаюсь сделать отчё ти персона

Stellareign committed:
qq

Stellareign committed:
Пытаюсь сделать отчё ти персона

Stellareign committed:
Merge remote-tracking branch 'origin/feture-TimurA' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/entity/shelters/CatsShelter.java
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

Stellareign committed:
Отчёты и персоны записываются БД, но не создаются - записываются пустые, блоки свитча не отрабатывают :(.

alkor63 committed:
Передвинул всю реализацию из Сервиса в слой Репозиторий

Stellareign committed:
Отчёты и персоны записываются БД, но не создаются, при определённых условиях записываются пустые, блоки свитча не отрабатывают :(.

Stellareign committed:
В сохранении отчёта убран блок else, так как будем делать проверку полноты отчётов по null значениям

Stellareign committed:
В сохранении отчёта убран блок else, так как будем делать проверку полноты отчётов по null значениям

alkor63 committed:
Сделал цепочку Контроллер-Сервис-Репозиторий для отчетов OwnerReport
есть проблема: в классе Volunteer в строке 35 ВСЕГДА! ownerReportService = null

alkor63 committed:
Написал Волонтеру метод проверки правильности заполнения отчета
рассмотрены все сочетания заполненных и пустых полей

Stellareign committed:
вернула обратно в кейсы отчёта break - всё то же самое.

Roman-1901 committed:
ww

alkor63 committed:
Делаю OwnerReport2

alkor63 committed:
Merge remote-tracking branch 'origin/feature-RomanN' into feature-AlexeyK

# Conflicts:
#	src/main/java/com/ward_n6/Controllers/OwnerController.java
#	src/main/java/com/ward_n6/Controllers/PetController.java
#	src/main/java/com/ward_n6/entity/people/Volunteer.java
#	src/main/java/com/ward_n6/entity/reports/OwnerReport.java
#	src/main/java/com/ward_n6/service/Impl/OwnerServiceImpl.java
#	src/main/java/com/ward_n6/service/Impl/PetServiceImpl.java
#	src/main/java/com/ward_n6/service/Impl/PetsOwnerArchiveServiceImpl.java
#	src/main/java/com/ward_n6/service/Impl/PetsOwnerServiceImpl.java
#	src/main/resources/application.properties

alkor63 committed:
перелопатил весь код, не могу избавиться от null в строке 35 класса Volunteer

Rumonz1 committed:
комментарии

elizaveta-popova committed:
Добавлены константы и тесты для класса infocontroller

elizaveta-popova committed:
Merge branch 'feature-AlexeyK' into feature-Elizaveta

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

alkor63 committed:
перелопатил весь код, не могу избавиться от null в строке 35 класса Volunteer
Через Сваггер методы контроллера отчётов работают
НО в Волонтере по прежнему OwnerReportRepository = null

elizaveta-popova committed:
Добавлены тесты для контроллеров Pet и Owner

Stellareign committed:
Merge remote-tracking branch 'origin/feature-Elizaveta' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/entity/owners/Owner.java
#	src/main/java/com/ward_n6/entity/owners/OwnerReport.java
#	src/main/java/com/ward_n6/listener/GiveReport.java
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java
#	src/main/java/com/ward_n6/repository/OwnerRepository.java
#	src/main/java/com/ward_n6/service/OwnerService.java
#	src/main/resources/liquibase/init.sql

Stellareign committed:
Merge remote-tracking branch 'origin/feature-AlexeyK' into feature-RuAna

# Conflicts:
#	pom.xml
#	src/main/java/com/ward_n6/entity/reports/OwnerReport.java
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

alkor63 committed:
Переименовал класс Volunteer в VolunteerService и переместил его в пакет Service
(типа поучаствовал в наведекнии порядка)
Доделал все методы (в т.ч. изменение/редактирование) в "подопытном" контроллере OwnerReportController
Для этого контроллера написал тесты. Все позитивные тесты работают, с негативными какие-то проблемы (вероятно связанные с контейнером Optional)
Пока тесты на Exceptions просто закомментил, потом буду разбираться

Stellareign committed:
Merge remote-tracking branch 'origin/feature-AlexeyK' into feature-RuAna

Stellareign committed:
Сделана часть тестов. Часть тестов закомментирована для запска приложения- тесты в разработке, требуется корректировка.
Проблема с запуском метода сохранения отчёта и овнера не решена.
В отчёте добавилось сохранение типа животного, но почему-то цифрой.

Rumonz1 committed:
Merge branch 'feature-RuAna' into feture-TimurA

# Conflicts:
#	src/main/java/com/ward_n6/entity/shelters/PetShelter.java
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

Stellareign committed:
Процесс доработки сохранения отчёта.
В пакете таймера добавлен пример из старого бота.

Stellareign committed:
Сохранение отчёта заработало!
Осталось фото и сам овнер.
И потом можно додумать, как быть с типом животного.

Stellareign committed:
Проблема с сохранением овнера

Roman-1901 committed:
CatsDogsBD

Roman-1901 committed:
CatsDogsBD

Stellareign committed:
Сохранение овнера работает, призведён рефакторинг классов обслуживания овнера и контроллера -> нужно проверить и исправить у Алексе.

Stellareign committed:
Вернула имплементацию интерфейса OwnerService и обратно переименовала класс OwnerServiceImpl, пока работает

Roman-1901 committed:
CatsDogsTest

Rumonz1 committed:
Merge branch 'feature-RuAna' into feture-TimurA

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

Stellareign committed:
Вернула имплементацию интерфейса OwnerService и обратно переименовала класс OwnerServiceImpl, пока работает

Stellareign committed:
В работе метод заполнения PetsOwnera.
1. Почему не проходит проверка на сущетствование овнера в БД?

alekseykoshelev commented on src/main/java/com/ward_n6/Controllers/OwnerReportController.java:
```diff
@@ -0,0 +1,92 @@
+            //удалено: || ownerReport.getOwnerId() == null
+            throw new InvalidRequestException("PatientRecord or ID must not be null!");
+        }
+        Optional optionalOwnerReport = ownerReportRepository.findById(ownerReport.getOwnerId());
```
Тут можно использовать дженерик. `Optional<OwnerReport> ` и не нужно будет делать каст ниже

Stellareign replied:
+

alekseykoshelev commented on src/main/java/com/ward_n6/Controllers/OwnerReportController.java:
```diff
@@ -0,0 +1,92 @@
+        }
+        OwnerReport existingOwnerReport = (OwnerReport) optionalOwnerReport.get();
+
+        existingOwnerReport.setHavePhoto(ownerReport.isHavePhoto());
```
Эту логику лучше перенести в сервис

Stellareign replied:
+

alekseykoshelev commented on src/main/java/com/ward_n6/entity/owners/Owner.java:
```diff
@@ -0,0 +1,66 @@
+    @javax.persistence.Id
+    @Column(name = "owner_id", nullable = false)
+
+    private Long ownerId;
```
Примарные ключи лучше все таки называть как `id`

Stellareign replied:
+

alekseykoshelev commented on src/main/java/com/ward_n6/repository/Impl/PetsOwnerArchiveRepositoryImpl.java:
```diff
@@ -0,0 +1,74 @@
+import java.util.Map;
+
+@Repository
+public class PetsOwnerArchiveRepositoryImpl implements PetsOwnerArchiveRepository {
```
Этот репозиторий выглядит лишним, мы ведь храним все в БД

Stellareign replied:
+

alekseykoshelev commented on src/main/java/com/ward_n6/repository/PetsOwnerArchiveRepository.java:
```diff
@@ -0,0 +1,24 @@
+
+import java.util.List;
+
+public interface PetsOwnerArchiveRepository {
```
Этот репозиторий не должен расширять JpaRepository?

Stellareign replied:
+

alekseykoshelev commented on src/main/java/com/ward_n6/service/OwnerServiceImpl.java:
```diff
@@ -0,0 +1,186 @@
+@Repository
+public class OwnerServiceImpl implements com.ward_n6.service.interfaces.OwnerService {
+    private final OwnerRepository ownerRepository;
+    private Map<Integer, Owner> ownerMap = new HashMap<>();
```
Мапа выглядит лишней, ведь теперь все храним в базе

Stellareign replied:
+

alekseykoshelev commented on src/main/java/com/ward_n6/service/PetServiceImpl.java:
```diff
@@ -0,0 +1,140 @@
+    private CatRepository catRepository;
+    private DogRepository dogRepository;
+
+    private Map<Integer, Pet> petMap = new HashMap<>();
```
Тут аналогично

Stellareign replied:
+

Stellareign committed:
Небольшие правки по замечаниям: удалены ненужные классы и пакеты

Rumonz1 committed:
222

Rumonz1 committed:
Merge branch 'feature-RuAna' into feture-TimurA

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

Stellareign committed:
Бот перестаёт отвечать после первого сообщения :(

Stellareign committed:
Ситуация с работой бота исправлена

Stellareign committed:
Внесены правки по замечаниям.

Stellareign committed:
Исправлен OwnerReportController

Stellareign committed:
Подправлены классы (убрана краснота) OwnertController
Pet
Owner
Volunteer

Stellareign committed:
изменена аннотация в Pet (@Builder)

Roman-1901 committed:
CatsDogsTest2

Roman-1901 committed:
CatsDogsTest2

alekseykoshelev commented on src/main/java/com/ward_n6/listener/PhotoHandler.java:
```diff
@@ -0,0 +1,71 @@
+
+                } catch (MalformedURLException e) {
+                    throw new RuntimeException(e);
+                } catch (FileNotFoundException e) {
```
Тут можно сделать один catch блок, объединив эксепшены через |

alekseykoshelev commented on src/main/java/com/ward_n6/entity/Photo.java:
```diff
@@ -0,0 +1,46 @@
+
+    @Lob
+    @Column(name = "photo", nullable = false)
+    PhotoSize photo;
```
Хорошо бы добавить модификаторы тут и ниже

alekseykoshelev commented on src/main/java/com/ward_n6/service/PetsOwnerArchiveServiceImpl.java:
```diff
@@ -0,0 +1,54 @@
+
+    @Override
+    public PetWithOwner editArchivePetWithOwnerById(int recordId, PetWithOwner petWithOwner) throws EditMapException {
+        return null;
```
Что-то много пустых методов в этом классе.

Stellareign replied:
А здесь Алексей ждём меня с фото и с petsOwner :-|

Roman-1901 committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RomanN

Rumonz1 committed:
просто пуш

Rumonz1 committed:
Merge branch 'feature-RuAna' into feture-TimurA

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java
#	src/main/java/com/ward_n6/repository/Impl/PetsOwnerArchiveRepositoryImpl.java
#	src/main/java/com/ward_n6/service/VolunteerService.java

Rumonz1 committed:
мёрж актуальной версии

Stellareign committed:
Правка по замечаниям:
PhotoHandler.java - эксепшены объеденены
Photo.java - добавлены модификаторы

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RomanN' into feature-RuAna

Stellareign committed:
Merge remote-tracking branch 'origin/feture-TimurA' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

Stellareign committed:
merge Рома, Тимур

Stellareign committed:
Тотальный merge

Stellareign committed:
Созданы строковые константы для листнера, вынесены в отдельный класс, рефакторинг листнеры по сообщениям бота - замены на константы.

Stellareign committed:
Созданы строковые константы для листнера, вынесены в отдельный класс, ревакторинг листнеры по сообщениям бота - замены на константы.

Stellareign committed:
Исправлены Петы: контроллер, сущности, тесты - совместно с Ромой. Сваггер работает.

Stellareign committed:
Исправлены Петы: контроллер, сущности, тесты - совместно с Ромой. Сваггер работает.

Stellareign committed:
Исправлены Петы: контроллер, сущности, тесты - совместно с Ромой. Сваггер работает.
Добавлен класс текстоввых констант для листнера - всё большае тексты заменены на константы.
Работа над добавлением из бота PetsOwner и Фото идёт.

Stellareign committed:
Исправлены Петы: контроллер, сущности, тесты - совместно с Ромой. Сваггер работает.
Добавлен класс текстоввых констант для листнера - всё большае тексты заменены на константы.
Работа над добавлением из бота PetsOwner и Фото идёт.

Stellareign committed:
Актуализирован таймер

Stellareign committed:
Актуализирован таймер

Stellareign committed:
Актуализирован таймер

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/listener/TelegramBotPetShelterUpdatesListener.java

Stellareign committed:
Все методы из листерена перенесены

Stellareign committed:
Все методы из листерена перенесены, рефакторинг класса

Stellareign committed:
петсОвнер и фото ещё не доделаны. Остальноу всё работает

Stellareign committed:
Сохранение petsOwner работает. Не работают проверки на ownerId у питомцев

Stellareign committed:
Сохранение petsOwner работает. Не работают проверки на ownerId у питомцев

Stellareign committed:
petsOwner работает
photo - почти работает - фото загружается, сохраняется в БД, не работает true/false в отчёте

Stellareign committed:
Нет связи фото и отчёта.

Stellareign committed:
Все кнопки работают.
В принципе, ботом можно пользоваться :)

Stellareign committed:
Кнопки работают, фото отражается отдельным отчётом в базе отчётов.

Stellareign committed:
Правки  \n

Stellareign committed:
Исправлены проверки при создании отчёта

Stellareign committed:
заметки

Stellareign committed:
Добавлены поля заполнения питомцев через сваггер - для удобства.
На остальное нет сил.

Stellareign committed:
Merge branch 'feature-RuAna' into dev

Stellareign committed:
Добавлены правки Лизы по овнеру

Stellareign committed:
Merge branch 'feature-RuAna' into dev

Stellareign committed:
Добавлены правки Лизы по овнеру

Stellareign committed:
Merge branch 'feature-RuAna' into dev

Stellareign committed:
Отменила изменения в контроллерах, пока не разобралась, почему работают некорректно

Stellareign committed:
Merge branch 'feature-RuAna' into dev

# Conflicts:
#	src/test/java/com/ward_n6/listener/ChatMessagerTest.java
#	src/test/java/com/ward_n6/listener/handlers/OwnerHandlerTest.java
#	src/test/java/com/ward_n6/listener/handlers/PhotoHandlerTest.java

Stellareign committed:
Пока безуспешно тестирую OwnerHandler

Stellareign committed:
Актуализированы классы OwnerController и тестовые по олонтёру и овнеру

Stellareign committed:
тесты

Stellareign committed:
Актуальная версия - не работают тесты OwnerReportController

Stellareign committed:
Актуальная версия - не работают тесты petsOwnerController

Stellareign committed:
Актуальная версия - не работают тесты OwnerReportController

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RuAna

# Conflicts:
#	src/main/java/com/ward_n6/service/interfaces/OwnerReportService.java
#	src/test/java/com/ward_n6/Controllers/OwnerReportControllerTest.java

Stellareign committed:
Merge branch 'feature-RuAna' into dev

Stellareign committed:
Немного изменила логику по методам OwnerReport (сервис и Controller: правка и добавление), написала все тесты по репортКонтроллер - вроде работают, написала тесты по контроллерам петов, вроде тоже все зелёные

Stellareign committed:
Ещё два метода для овнерСервис

Stellareign committed:
Немного подправила овнерКонтроллер

Stellareign committed:
Исправила тест по добавлению собаки.

Stellareign committed:
Merge branch 'feature-RuAna' into dev

Stellareign committed:
Добавлены тесты овнерСервис (Лиза), овнерКонтроллер (Алексей), откорректироаны, добавлен тест петсОвнерСервис

Stellareign committed:
Merge remote-tracking branch 'origin/feature-RuAna' into feature-RuAna

Stellareign committed:
Merge branch 'feature-RuAna' into dev

Stellareign committed:
Несколько штрихов в хэндлерах

Stellareign committed:
Добавила пару тестов

Stellareign committed:
Добавила пару тестов

Stellareign committed:
Удалила класс  и интерфейс PetService за ненадоностью

Stellareign committed:
Удалила два класса эксепшенов

Stellareign committed:
Удалила класс черновиков, все методы из него перенесла вмтекстовый файл

Stellareign committed:
несколько тестов на репозитории для покрытия

Stellareign committed:
несколько тестов на репозитории для покрытия

Stellareign committed:
Merge remote-tracking branch 'origin/release-14.09.23' into release-14.09.23

# Conflicts:
#	src/test/java/com/ward_n6/service/OwnerServiceImplTest.java

Stellareign committed:
+ пара тестов овнерРепортСервис на ошибку

Stellareign committed:
Добавлены тесты по ChatMessager

Stellareign committed:
Добавлены тесты по ChatMessager

Stellareign committed:
Подправила DateTimeFormatter в петах.

Stellareign committed:
Удалила класс PetWithOwner - не был задействован (вместо него PetsOwner), переместила классы с фото в пакеты отчётов

Stellareign committed:
Удалила класс PetWithOwner - не был задействован (вместо него PetsOwner), переместила классы с фото в пакеты отчётов

Stellareign committed:
Исправила небольшие косяки в отчёте: добавила дополнительные флаги верификации питомцев для отчёта.

Stellareign committed:
небольшие правки по сообщения от бота

Stellareign committed:
Удалила классы CatOwner и DogOwner (не были задействованы), создала файл README.
Кто хочет дополнить README - милости прошу))

Stellareign committed:
Ещё пара штрихов по фото.

Stellareign committed:
Удалила лишний файл от одного из прошлых проектов

Stellareign committed:
Create README.md

добавлен ридми

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Удалила лишний файл от одного из прошлых проектов

Stellareign committed:
Update README.md

Stellareign committed:
Merge remote-tracking branch 'origin/release-16.09.23' into release-16.09.23

Stellareign committed:
readme update

Stellareign committed:
Исправления проверки ID овнера при бронировании в записи БД в DogOwnerHandler и CatOwnerHandler

Stellareign committed:
Пара пробелов, readme update

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
Update README.md

Stellareign committed:
провела небольшой рефакторинг хэндлеров, вынесла некотрые кейсы в отдельные методы

Stellareign committed:
Merge branch 'release-18.09.23' into feature-RuAna
